src/mappings/nfts/setAttribute.ts

```diff
@@ -21,7 +22,12 @@ export async function handleAttributeSet(context: Context): Promise<void> {
   }
 
   if ('recipient' in final && event.trait === 'recipient') {
-    final.recipient = final.recipient ?? event.value as string
+    try {
+      final.recipient = final.recipient ?? addressOf(event.value as string)
+    } catch (error) {
+      console.log(error)
+      final.recipient = final.recipient ?? (event.value as string)
+    }
```

Comment:
btw this part is not looking really good since I wrapped it in a try catch to not have problem during indexing

Reply:
note that I tried to setup arrowsquid on my side too and it wasn't working ;(

```log
sqd:processorRpcError: Client error: Api called for an unknown Block: State already discarded for 0x6f7aebaa2e55d2b0ea6bd61d15cdc5b69cc22a3b1b2378bfdb165c2fedf9e312 at RpcClient.receiveResult (/squid/node_modules/@subsquid/rpc-client/lib/client.js:272:23) at /squid/node_modules/@subsquid/rpc-client/lib/client.js:194:29 at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Reply:
> my side too and it wasn't working ;(

Havent run yet, we will see